src/sparse/KokkosSparse_sptrsv_handle.hpp

```diff
@@ -109,26 +143,111 @@ class SPTRSVHandle {
   bool lower_tri;
 
   bool symbolic_complete;
+  bool numeric_complete;
 
   SPTRSVAlgorithm algm;
 
   int team_size;
   int vector_size;
 
+#ifdef KOKKOSKERNELS_ENABLE_SUPERNODAL_SPTRSV
+  // stored either in CSR or CSC
+  bool col_major;
+
+  // number of supernodal columns
+  signed_integral_t nsuper;
+
+  // etree, parent's id for each supernodal column
+  integer_view_host_t etree_host;
+
+  // dag
+  host_graph_t dag_host;
+
+  // map from supernode to column id, i.e., superdols[s] = the first column id of s-th supernode
+  integer_view_host_t supercols_host; // on the host
+  integer_view_t      supercols;      // on the default host/device
+
+  // workspace size
+  signed_integral_t lwork;
+  workspace_t work;
+  // offset to workspace for each supernodal column
+  integer_view_host_t work_offset_host;
+  integer_view_t      work_offset;
+
+  // 
+  bool merge_supernodes;
+  bool invert_offdiagonal;
+  int *etree;
+
+  // type of kernels used at each level
+  int sup_size_unblocked;
+  int sup_size_blocked;
+  integer_view_host_t diag_kernel_type_host;
+  integer_view_t      diag_kernel_type;
+  integer_view_host_t kernel_type_host;
+  integer_view_t      kernel_type;
+
+  // permutation
+  bool perm_avail;
+  integer_view_t perm;
+
+  // graphs
+  host_graph_t original_graph_host; // graph on host before merge (only if merged)
+  host_graph_t graph_host; // mirror of graph on host
+  graph_t graph;
+
+  // crsmat
+  crsmat_t crsmat;
+
+  // for supernodal spmv
+  bool spmv_trans;
+  crsmat_list_t sub_crsmats;
+  crsmat_list_t diag_blocks;
+
+  int num_streams;
+  #if defined(KOKKOS_ENABLE_CUDA)
+  cudaStream_t *cuda_streams;
+  #endif
+
+  // verbose
+  bool verbose;
+#endif
+
 public:
 
-  SPTRSVHandle ( SPTRSVAlgorithm choice, const size_type nrows_, bool lower_tri_, bool symbolic_complete_ = false ) :
+  SPTRSVHandle ( SPTRSVAlgorithm choice, const size_type nrows_, bool lower_tri_, bool symbolic_complete_ = false, bool numeric_complete_ = false ) :
     level_list(),
```

Comment:
`Kokkos::View` default initialization will do this for you.  You don't need to explicitly default-construct `Kokkos::View` like this.